packages/flutter/lib/src/material/tabs.dart

```diff
@@ -159,11 +162,16 @@ class Tab extends StatelessWidget implements PreferredSizeWidget {
       label = icon!;
     } else {
       calculatedHeight = _kTextAndIconTabHeight;
+      final EdgeInsetsGeometry effectiveIconMargin = iconMargin ?? (
+        Theme.of(context).useMaterial3
+          ? const EdgeInsets.only(bottom: 2)
+          : const EdgeInsets.only(bottom: 10)
+      );
```

Comment:
These are typically defined by tokens that we import directly from the material design spec, you can see some of them below in the M2 and M3 defaults. 
I can see this value is listed in https://m3.material.io/components/tabs/specs, @TahaTesser does this mean there should be a token available for this now?

Reply:
I knew about the tokens, but I wasn't able to find the margin in the spec. And I still can't... :sweat_smile:
Under which categories in the tree-view is it, and what is the ID of the token?

Reply:
> I can see this value is listed in https://m3.material.io/components/tabs/specs, @TahaTesser does this mean there should be a token available for this now?

Yes, `TabBar` and many widgets are missing tokens in Flutter for a while such as https://github.com/flutter/flutter/issues/124965

It was discuss that we shouldn't let the missing tokens get in the way, we can hard-code values in the template class which can be replaced in the future when the tokens area available.

Reply:
@davidmartos96 
Instead of hard-coding the value these values in the `effectiveIconMargin` variable when `iconMargin` is null. Please consider moving these values to the template class and place `iconMargin` as static value. This will keep the defaults values in a single place and make easier to update with tokens later.